OpenTelemetry.AutoInstrumentation.proj

```diff
@@ -14,16 +14,15 @@
     <TestApplicationProject Include="test\test-applications\integrations\dependency-libs\*\*.csproj" />
     <TestApplicationProjectFX Include="test\test-applications\integrations\aspnet\*\*.csproj" Condition="'$(OS)' == 'Windows_NT'" />
     <MockProject Include="test\test-applications\mocks\*\*.csproj" />
-    <IntegrationProject Include="test\integration-tests\*\*.csproj" />
-    <IntegrationProjectFX Include="test\integration-tests\aspnet\*\*.csproj" Condition="'$(OS)' == 'Windows_NT'" />
+    <IntegrationProject Include="test\IntegrationTests\IntegrationTests.csproj" />
```

Comment:
Just curios to know why this is not a part of `Restore`/`Clean` target?

Reply:
Already covered with `UnitTestProject`